tutorials/darcy_thermo_mech/doc/content/workshop/numerical/fvm_overview.md

```diff
@@ -0,0 +1,245 @@
+# New Feature: Finite Volume Method (FVM)
+
+!---
+
+## Advantages of Finite Volume Method
+
++(1) Flexible:+ Works with arbitrary cell types (even if they are mixed in the same problem)
```

Comment:
We currently don't support polyhedra, which is the standard for FV. Could add between parenthesis the current cell support?

Reply:
To be honest, I am not sure we can deal with mixed types either. This is just a list of theoretical advantages of using FVM. We could probably considerably shrink this list if I considered the details of the current implementation in MOOSE.